packages/react/src/components/Toggle/next/Toggle.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright IBM Corp. 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { settings } from 'carbon-components';
+import { useControllableState } from '../../../internal/useControllableState';
+
+const { prefix } = settings;
+
+export function Toggle({
+  className,
+  defaultToggled = false,
+  disabled = false,
+  hideLabel = false,
+  id,
+  labelA = 'Off',
+  labelB = 'On',
+  labelText,
+  onClick = () => {},
```

Comment:
We should update thse types of props to not have a default value and just check for their existence in the handlers

```suggestion
  onClick,
```

Reply:
👍